src/main/java/com/easypost/exception/General/InvalidParameterError.java

```diff
@@ -0,0 +1,24 @@
+package com.easypost.exception.General;
+
+import com.easypost.exception.EasyPostException;
+
+public class InvalidParameterError extends EasyPostException{
+    /**
+     * InvalidParameterError constructor.
+     *
+     * @param message the exception message
+     */
+    public InvalidParameterError(final String message) {
+        super(message);
+    }
+
+    /**
+     * constructor.
+     *
+     * @param message the exception message
+     * @param e the Throwable object
+     */
+    public InvalidParameterError(final String message, Throwable e) {
```

Comment:
Question: Why do some of these exceptions have a Throwable constructor and others don't?
I understand what the Throwable does (basically allows for inner-nested exceptions), why are we inconsistent with which exception types can and can't have an inner Throwable?

Reply:
@jchen293 care to answer? Do we need to correct this? Pinging to make sure it doesn't fall through the cracks.

Reply:
We have a `Throwable` in this Exception class because there is a `UnsupportedEncodingException` that could possibly throw when encoding the report type, you could find the code here: https://github.com/EasyPost/easypost-java/blob/de759f2bad6f8f608591aa15a42bd0fd17ad7215/src/main/java/com/easypost/model/Report.java#L211

Reply:
@Justintime50 Had to merge first so I grab get the link to the latest code commit